meshtastic/node_runtime/response_runtime.py

```
@@ -0,0 +1,423 @@
+    def _format_and_emit_enum(
+        self,
+        *,
+        field_name: str,
+        value: int,
+        enum_type: Any,
+    ) -> None:
+        """Emit enum metadata field using enum name when valid and raw value otherwise."""
+        if value in enum_type.values():
+            formatted = enum_type.Name(value)
+        else:
+            formatted = value
```

Ruff (0.15.7)

[warning] Use ternary operator `formatted = enum_type.Name(value) if value in enum_type.values() else value` instead of `if`-`else`-block

Replace `if`-`else`-block with `formatted = enum_type.Name(value) if value in enum_type.values() else value`

(SIM108)